fchost/fchost.go

```diff
@@ -17,11 +17,13 @@ var (
 	log = logging.Logger("fchost")
```

Comment:
Mostly start documenting methods. This was done not only on this file, but in mostly all ones.